eofConvention.fsx

```diff
@@ -0,0 +1,30 @@
+open System.IO
+
+let EolAtEof(path: FileInfo) = 
+
+    use streamReader = new StreamReader (path.FullName)
+    let filetext = streamReader.ReadToEnd()
+    
+    if filetext <> "" then
```

Comment:
@realmarv nit: use String.Empty instead of `""`

Reply:
Done